spongeauth/accounts/models.py

```diff
@@ -49,6 +49,11 @@ def validate_username(username):
         raise ValidationError(errs)
 
 
+def validate_discord_id(discord_id):
+    if not re.match(r'^(.*)#(\d{4})$', discord_id):
```

Comment:
This should probably be at least .+, because we shouldn't permit `#1234` as a Discord ID. Probably?